diff -Naur qucs-0.0.17.orig/qucs-core/src/module.cpp qucs-0.0.17/qucs-core/src/module.cpp
--- qucs-0.0.17.orig/qucs-core/src/module.cpp	2013-06-23 07:59:37.000000000 +0200
+++ qucs-0.0.17/qucs-core/src/module.cpp	2014-02-05 18:49:56.000000000 +0100
@@ -378,7 +378,7 @@
 /* The function emits a complete list of the registered component
    definitions as compilable C-code. */
 void module::print (void) {
-  fprintf (stdout, def_prefix);
+  fprintf (stdout, "%s", def_prefix);
   qucs::hashiterator<module> it;
   for (it = qucs::hashiterator<module> (modules); *it; ++it) {
     module * m = it.currentVal ();
@@ -393,8 +393,8 @@
 	     printstr (def->type), def->nodes, def->action, def->substrate,
 	     def->nonlinear, def->type, def->type);
   }
-  fprintf (stdout, def_stop);
-  fprintf (stdout, def_start);
+  fprintf (stdout, "%s", def_stop);
+  fprintf (stdout, "%s", def_start);
   fprintf (stdout, "{\n");
   for (it = qucs::hashiterator<module> (modules); *it; ++it) {
     module * m = it.currentVal ();
@@ -403,6 +403,6 @@
   }
   fprintf (stdout, "  def_End\n");
   fprintf (stdout, "};\n");
-  fprintf (stdout, def_suffix);
+  fprintf (stdout, "%s", def_suffix);
 }
 #endif /* DEBUG */
diff -Naur qucs-0.0.17.orig/qucs-core/src/converter/qucs_producer.cpp qucs-0.0.17/qucs-core/src/converter/qucs_producer.cpp
--- qucs-0.0.17.orig/qucs-core/src/converter/qucs_producer.cpp	2013-06-23 07:59:37.000000000 +0200
+++ qucs-0.0.17/qucs-core/src/converter/qucs_producer.cpp	2014-02-05 19:07:39.010000000 +0100
@@ -499,7 +499,7 @@
   
   sprintf (txt, "\n<Component %s>\n", def->instance[0] == dev->stype[0] ?
 	   &def->instance[1] : def->instance);
-  fprintf (qucs_out, txt);
+  fprintf (qucs_out, "%s", txt);
   fprintf (qucs_out, "  <Description>\n");
   fprintf (qucs_out, "  </Description>\n");
   fprintf (qucs_out, "  <Model>\n");
